tests/utils/test_paralog.py

```diff
@@ -36,3 +37,28 @@ def test_claiming():
         print(fp.read())
 
     shutil.rmtree('testing-paralog')
+
+def test_log_message():
+
+    ''' 
+    test that both normal print messages go to the log and log_message calls go to the log.
+    
+    This test creates files that have to be deleted for the test to run sucessfully again. The test deletes these files at the 
+    end, but if it fails before, the user should manually delete these files after fixing the test. 
+    '''
+
+    statman = paralog.StatusManager(jobname='test', jobtitle='Testing process', logdir='testing-paralog', timeout=60*60)
+    print("Printed message")
```

Comment:
is this print necessary ?

Reply:
Yup, this is testing the normal logging capabilities.